server/controllers/bookController.js

```diff
@@ -74,6 +74,49 @@ class bookController {
         error: error.message,
       });
     }
+  };
+  static async deleteBook(req, res){
+
+    try{
+       const bookId = req.params.bookId;
+       if(isNaN(bookId)){
+        return res.status(400).json({
+          status: 400,
+          error: 'Book id must be integer',
+        });
+       }
```

Comment:
Hi @hakoemmy  nice work 
I think this would have been done better with params checker (middleware) using Joi or any validator

Reply:
I got what you mean. Allow me to make changes